Scripts/SaneDocReport/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+- DOCX reports now fail gracefully (error shown in the docx instead of failure to generate completely).
+- Added the itemsSection element to DOCX reports (mainly used in investigation reports).
```

Comment:
where is the relevant changes for this? i only see the error handling.

Reply:
It is the internal docker (the tag change...)